<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>基准股票</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="basicstock"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>股票池</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="stockpool"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>最小相关度</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="minValue">
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>最大相关度</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="maxValue">
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QTableWidget" name="table"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::BottomToTop</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="button">
     <property name="text">
      <string>计算</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
